daemon/commit.go

```diff
@@ -235,18 +245,15 @@ func (daemon *Daemon) Commit(name string, c *backend.ContainerCommitConfig) (str
 }
 
 func (daemon *Daemon) exportContainerRw(container *container.Container) (io.ReadCloser, error) {
-	if err := daemon.Mount(container); err != nil {
```

Comment:
Why does this function no longer mount the layer?

Reply:
We assumed that `TarStream()` should mount the layer if needed. ( see https://github.com/moby/moby/pull/30696#discussion_r117799262 for more information )

But I just noticed that the `Diff()` function for windows requested that the layer should be mounted when calling this function. ( see https://github.com/moby/moby/blob/master/daemon/graphdriver/windows/windows.go#L474 )
So, is this the reason why CI fails all the time?

Reply:
@johnstep Thanks for finding this. I have modified the codes and now CI on windows machine finally passed but CI for powerpc failed due to a full disk.